README.md

```diff
@@ -49,15 +49,19 @@ notify-pass credentials/mmg
 
 ### Postgres
 
-Install [Postgres.app](http://postgresapp.com/).
+Postgres is built in the CONCOURSE_TESTS as an extra layer derived from the TEST layer.
+This is built to allow the tests to run using the API PRODUCTION layer.
```

Comment:
I don't know if this comment change is that helpful.

It used to highlight the fact that this app has a dependency on Postgres which needs to be managed manually. Now that context is deleted and replaced with a comment about a very specific setup for testing via Concourse (which I think is the main thing we're talking about in this PR?).

```suggestion
This app requires Postgres to run.

If you are using [notifications-local](https://github.com/alphagov/notifications-local), the correct Postgres version will be provided automatically by the docker-compose file.

If you are running this app manually, you should will need to manage Postgres yourself. Install [Postgres.app](http://postgresapp.com/). Check the docker-compose file above to find the correct Postgres version to use.

When running our test suite, Postgres is baked into the test image under the `concourse_tests` step of `docker/Dockerfile`.
```

Reply:
postgres is built within the image as an extra layer. Running locally or in concourse, postgres will be present in the "concourse_tests" layer. 
There is no need to manually install postgres either way.
Not sure I understand your comments.

Reply:
please have a look at the original task:
https://trello.com/c/SRDI5dT8/576-get-api-unit-tests-to-run-in-the-api-docker-image-not-the-generic-unit-test-image-on-concourse

Reply:
Yes, but you're thinking about testing only.

The readme is documenting the general app altogether - ie what you need to run the app, not what you need to _test_ the app.

The readme currently says that if you want to _run_ the app, you need to install Postgres manually. That's still true so I don't think we should change the readme to only talk about the testing side of things.

Reply:
I agree with Sam here. I think this section should remain about what you need to do to get Postgres so you can run the app locally.

Any additional context about running the unit tests in Concourse isn't too important for the README. If we want to talk about running the unit tests locally in docker, then that would probably be best in the section 'To test the application' which is below.